Source/ChocolateyGui/Properties/Resources.nb.resx

```diff
@@ -0,0 +1,553 @@
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="Argument_cant_be_null_or_empty" xml:space="preserve">
+    <value>Argumentet kan ikke være null eller tomt.</value>
+  </data>
+  <data name="ChocolateyDialog_ConsoleOutput" xml:space="preserve">
+    <value>Konsolltekst</value>
+  </data>
+  <data name="Controls_PackageListByAuthor" xml:space="preserve">
+    <value>Av: </value>
+  </data>
+  <data name="Controls_PackageListDownloads" xml:space="preserve">
+    <value>nedlastinger</value>
+  </data>
+  <data name="InvalidVersionString" xml:space="preserve">
+    <value>Ugyldig versjonsstreng.</value>
+  </data>
+  <data name="LocalSourceView_ButtonExport" xml:space="preserve">
+    <value>Eksporter</value>
+  </data>
+  <data name="LocalSourceView_ButtonRefreshPkgs" xml:space="preserve">
+    <value>Oppdater Pakkeliste</value>
+  </data>
+  <data name="LocalSourceView_ButtonUpdateAll" xml:space="preserve">
+    <value>Oppdater Alle</value>
+  </data>
+  <data name="LocalSourceView_CheckboxPkgsWithUpdates" xml:space="preserve">
+    <value>Vis Kun Pakker med Oppdateringer</value>
+  </data>
+  <data name="LocalSourceView_Grid_Authors" xml:space="preserve">
+    <value>Forfattere</value>
+  </data>
+  <data name="LocalSourceView_Grid_LatestVersion" xml:space="preserve">
+    <value>Siste Versjon</value>
+  </data>
+  <data name="LocalSourceView_Grid_Name" xml:space="preserve">
+    <value>Navn</value>
+  </data>
+  <data name="LocalSourceView_Grid_Version" xml:space="preserve">
+    <value>Versjon</value>
+  </data>
+  <data name="LocalSourceView_SearchBoxText" xml:space="preserve">
+    <value>Søk:</value>
+  </data>
+  <data name="PackageView_Authors" xml:space="preserve">
+    <value>Forfattere:</value>
+  </data>
+  <data name="PackageView_ButtonInstall" xml:space="preserve">
+    <value>Installer</value>
+  </data>
+  <data name="PackageView_ButtonReinstall" xml:space="preserve">
+    <value>Reinstaller</value>

```

Comment:
Nei, det er vel egentlig ikke ett norsk ord. Men ett ord jeg har sett my brukt uansett.
'Installere på nytt' blir vel kanskje litt for langt med tanke på at teksten er på en knapp, alternativet kunne kanskje være å bruke Ominstaller?